net/mosquitto/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; indent-tabs-mode: nil -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
```

Comment:
See https://trac.macports.org/ticket/47197 and determine if the following line can be added:
`cmake.out_of_source yes `